src/pcapi/tasks/decorator.py

```diff
@@ -31,6 +32,9 @@ def delay(payload: payload_in_kwargs):
                 f(payload)
                 return
 
+            if issubclass(payload.__class__, BaseModel):
```

Comment:
(détail / style)

C'est un cas où je préfère importer le module `pydantic` plutôt que directement `BaseModel`. Car, à la lecture, on ne sait pas ce qu'est `BaseModel`, ici. Si on importe `pydantic` plus haut, on peut écrire :

```suggestion
            if issubclass(payload.__class__, pydantic.BaseModel):
```

... ce qui me semble bien plus clair.